src/AppKit/ActionDispatcher.cs

```diff
@@ -113,5 +113,10 @@ public bool ValidateMenuItem (NSMenuItem menuItem)
 			return true;
 		}
 
+		public bool WorksWhenModal
```

Comment:
that likely (98%) needs to be preserved or the linker will remove it
if not (2%) then it needs a test, because that behaviour could change

Reply:
Good catch. I'll look at testing it locally shortly.